security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -12,6 +12,70 @@ These release notes track the development of {cert-manager-operator}.
+=== New features and enhancements
+
+This is the general availability (GA) release of the {cert-manager-operator}.
+
+[id="cert-manager-log-level-1.11.1"]
+==== Setting log levels for cert-manager and the {cert-manager-operator}
+* To troubleshoot issues with cert-manager and the {cert-manager-operator}, you can now configure the log level verbosity by setting a log level for cert-manager and the {cert-manager-operator}. For more information, see xref:../../security/cert_manager_operator/cert-manager-log-levels.adoc#cert-manager-log-levels[Configuring log levels for cert-manager and the {cert-manager-operator}]
+
+[id="cert-manager-authentication-aws-1.11.1"]
+==== Authenticating the {cert-manager-operator} with AWS
+* You can now configure cloud credentials for the {cert-manager-operator} on AWS clusters with Security Token Service (STS) and without STS.
+
+[id="cert-manager-authentication-gcp-1.11.1"]
+==== Authenticating the {cert-manager-operator} with GCP
+* You can now configure cloud credentials for the {cert-manager-operator} on GCP clusters with Workload Identity and without Workload Identity.
+
+[id="cert-manager-dns01-acme-aws-1.11.1"]
+==== Configuring an ACME issuer to solve DNS01 challenges by using ambient credentials on AWS
+* You can use the {cert-manager-operator} to set up an ACME issuer to solve DNS-01 challenges by using ambient credentials on AWS.
+
+[id="cert-manager-dns01-acme-gcp-1.11.1"]
+==== Configuring an ACME issuer to solve DNS01 challenges on GCP
+* You can use the {cert-manager-operator} to set up an ACME issuer to solve DNS-01 challenges by using ambient and explicit credentials on GCP.
+
+[id="cert-manager-dns01-acme-azure-1.11.1"]
+==== Configuring an ACME issuer to solve DNS01 challenges on Microsoft Azure
+* You can use the {cert-manager-operator} to set up an ACME issuer to solve DNS-01 challenges by using explicit credentials on Microsoft Azure.
+
+[id="cert-manager-operator-1.11.1-bug-fixes"]
+=== Bug fixes
+
+* Previously, the `cm-acme-http-solver` pod did not use the latest published Red Hat image `registry.redhat.io/cert-manager/jetstack-cert-manager-acmesolver-rhel9`. With this release, the `cm-acme-http-solver` pod uses the latest published Red Hat image `registry.redhat.io/cert-manager/jetstack-cert-manager-acmesolver-rhel9`. (link:https://issues.redhat.com/browse/OCPBUGS-10821[*OCPBUGS-10821*])
+
+* Previously, the {cert-manager-operator} did not support changing labels for cert-manager pods such as controller, CA injector, and Webhook pods. With this release, you can add labels to cert-manager pods. (link:https://issues.redhat.com/browse/OCPBUGS-8466[*OCPBUGS-8466*])
+
+* Previously, the `operatorLogLevel` field did not update the log verbosity level in the {cert-manager-operator}. Now, the `operatorLogLevel` field properly updates the log verbosity level in the {cert-manager-operator}. (link:https://issues.redhat.com/browse/OCPBUGS-9994[*OCPBUGS-9994*])
```

Comment:
operatorLogLevel field does not work for this. OPERATOR_LOG_LEVEL does it.
```suggestion
* Previously, there is no way to update the log verbosity level in the {cert-manager-operator}. Now it is implemented using an environmental variable OPERATOR_LOG_LEVEL in its subscription resource. (link:https://issues.redhat.com/browse/OCPBUGS-9994[*OCPBUGS-9994*])
```

Reply:
Addressed. Changed the wording little bit